imagekitio/models/UpdateFileRequestOptions.py

```diff
@@ -11,16 +11,20 @@ def __init__(
         tags: List[str] = None,
         custom_coordinates: str = None,
         custom_metadata: json = None,
+        publish: json = None,
     ):
-        if remove_ai_tags is not None:
-            self.remove_ai_tags = remove_ai_tags
-        if webhook_url is not None:
-            self.webhook_url = webhook_url
-        if extensions is not None:
-            self.extensions = extensions
-        if tags is not None:
-            self.tags = tags
-        if custom_coordinates is not None:
-            self.custom_coordinates = custom_coordinates
-        if custom_metadata is not None:
-            self.custom_metadata = custom_metadata
+        if publish is not None:
+            self.publish = publish
+        else:
+            if remove_ai_tags is not None:
+                self.remove_ai_tags = remove_ai_tags
+            if webhook_url is not None:
+                self.webhook_url = webhook_url
+            if extensions is not None:
+                self.extensions = extensions
+            if tags is not None:
+                self.tags = tags
+            if custom_coordinates is not None:
+                self.custom_coordinates = custom_coordinates
+            if custom_metadata is not None:
+                self.custom_metadata = custom_metadata
```

Comment:
We shouldn't do an if else here, because "If any are present, an error will be returned: `Your request cannot contain any other parameters when publish is present`." this statement will become false, because you will always be sending just publish

Reply:
Ye checks SDK me mat dalo Bhai. Don't do api validation in SDKs.

Reply:
Ha updated it.
@imagekitio  we are just checking here if a property is not none then sending it error and other checks are handled by API